tools/crm_mon.c

```diff
@@ -1986,6 +1986,7 @@ crm_diff_update(const char *event, xmlNode * msg)
 static int
 mon_refresh_display(gpointer user_data)
 {
+    bool do_reset_finish = options.daemonize || (output_format == mon_output_html && out->dest != stdout);
```

Comment:
Would it make sense to reset if out->dest != stdout regardless of format?

Reply:
It would be worth a test, but I think it would be fine to do that.